Controllers/FileController.cs

```diff
@@ -1,20 +1,77 @@
 using Microsoft.AspNetCore.Mvc;
+using System.IO;
+using System.Linq;
+using System.Text.Json;
+using System.Threading.Tasks;
+using System.Collections.Generic; 
+using System.Globalization;
+
 namespace TodoApi.Controllers
 {
+    public class Author
+    {
+        public string Name { get; set; }
+        public string CountryOfOrigin { get; set; }
+        public DateTime LastSubmissionTimestamp { get; set; }
+    }
+    public class FormData
+    {
+        public string[]? Files { get; set; }
+        public Dictionary<string, int>? DessertVotes { get; set; }
+        public string? CurrentWeather { get; set; }
+        public List<Author>? AuthorsList { get; set; }
+    }
+
     [Route("api/[controller]")]
     [ApiController]
     public class FileController : ControllerBase
     {
+        
+        private const string FilePath = "formDataStorage.json";
+        private Dictionary<string, int>? dessertVotes = new Dictionary<string, int>();
+        private string? currentWeather = "clear";
+        private List<Author> authorsList = new List<Author> {};
+        private dynamic allData;
+
+        private async Task InitializeFormTypes()
+        {
+            if (System.IO.File.Exists(FilePath))
+            {
+                var json = await System.IO.File.ReadAllTextAsync(FilePath);
```

Comment:
This whole section seems like we should just deserialize the FormData and store that object as the instance